pyproject.toml

```diff
@@ -1,43 +1,91 @@
 [build-system]
 requires = [
-    "setuptools>=42",
     "wheel",
     "ninja",
-    "cmake>=3.12",
+    "cmake>=3.20",
     "pybind11>=2.10.4",
-    "numpy",
+    "numpy>=2",
     "PyYAML",
     "nanobind",
+    "delvewheel; platform_system=='Windows'",
+    "setuptools-scm>=8",
+    "scikit-build-core>=0.10,<0.11"
 ]
+build-backend = "scikit_build_core.build"
+
+[project]
+name = "finch-mlir"
+dynamic = ["version"]
+description = "Finch MLIR distribution as wheel."
+readme = "README.md"
+dependencies = [
+    "PyYAML>=6",
+    "numpy>=1.17",
+]
+
+[tool.setuptools_scm]
+
+[tool.scikit-build]
+cmake.source-dir = "llvm-project/llvm"
+ninja.make-fallback = false
+build-dir = "build"
+cmake.args = ["-G Ninja", "-UNB_SUFFIX", "-UNB_SUFFIX_S"]
+metadata.version.provider = "scikit_build_core.metadata.setuptools_scm"
+wheel.exclude = ["**/*", "!mlir_finch/**/*"]
```

Comment:
What does it mean? Is is possible to use some kind of `wheel.include` instead? This line is overly cryptic to me.

Reply:
Unfortunately not. This is `.gitignore` syntax. I'll add a comment here, but in short, lines coming later override the lines before. This says, essentially, that "ignore all files, except those in the `mlir_finch` directory".

Reply:
Comment added in https://github.com/finch-tensor/Finch-mlir/pull/16/commits/cc282d892244e6135641311d5212712c7174dcf6.